pending/windows_recycle_bins_more.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+    @url none
+    @cleanerversion v0.5.0
+    @cleanerdate 2019-03-11
+    @cleanerby https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-10
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
+-->
+<cleaner id="windows_recycle_bins_more" os="windows">
+  <label translate="true">Windows Recycle Bins - More</label>
+  <description>Deletes the Windows Recycle Bins of all drives from C to Z</description>
+  <option id="windows_recycle_bins_all">
```

Comment:
I would not repeat `windows_recycle_bins` in the option ID because it's in the cleaner ID. When cleaning on the command line, this can feel too verbose.

Reply:
I will fix that!